### PR TITLE
Douglas/finished camera with master

### PR DESCRIPTION
**Pull request by douglaswsilva**
Change thread for subscriptions.

### CODE REVIEW

File: Source/Dronecode-SDK-Swift/Dronecode-SDK-Swift/Camera.swift [hunk 1]
```diff
@@ -2,7 +2,93 @@ import Foundation
 import SwiftGRPC
 import RxSwift
 
+// MARK: - CameraResult
```
Comment (julianoes) [inline Source/Dronecode-SDK-Swift/Dronecode-SDK-Swift/Camera.swift]: The file has suddenly the executable bit set, see `100644 → 100755`. Please fix it using `chmod -x Camera.swift`.

File: Source/Dronecode-SDK-Swift/Dronecode-SDK-Swift/Mission.swift [hunk 2]
```diff
@@ -163,6 +163,8 @@ public class Mission {
 
             return Disposables.create()
         }
+        .subscribeOn(scheduler)
```
Comment (julianoes) [inline Source/Dronecode-SDK-Swift/Dronecode-SDK-Swift/Mission.swift]: See above, all files suddenly are executable!

**Comment by julianoes** [comment]
I suggest that you next time push directly to this repo and not to your fork. That way @JonasVautherin and I can add commits as well. You should have the needed permissions.

**Comment by julianoes** [comment]
To remove the executable fix from all swift files, just do:
```
find . -type f -perm +0111 -name "*.swift" | xargs chmod -x
```

**Comment by douglaswsilva** [comment]
Done. Let me know if it worked.

**Comment by julianoes** [comment]
@JonasVautherin there are now conflicts, not sure what to do.

File: Dronecode_SDK_SwiftIntegrationTests/ActionTest.swift [hunk 3]
```diff
@@ -5,16 +5,28 @@ import XCTest
...
 
-        action.arm().subscribe(onError: { error in XCTFail("\(error)") })
+        action.arm()
+            .subscribe(onCompleted: {
+                expectation.fulfill()
+            }, onError: { (error) in
+                XCTFail("\(error)")
+            })
+        
+        wait(for: [expectation], timeout: 10.0)
```
Comment (JonasVautherin) [inline Dronecode_SDK_SwiftIntegrationTests/ActionTest.swift]: Nice!

**Pull request merged**